packages/feedback-react/src/Feedback.tsx

```diff
@@ -1,47 +1,60 @@
+import { PrimaryButton, TertiaryButton } from "@fremtind/jkl-button-react";
+import { SuccessMessage } from "@fremtind/jkl-message-box-react";
 
-    return (
-        <FieldGroup legend={description} className="jkl-feedback__fieldset">
-            {options?.map((option) => {
-                const { label, value: optionValue } = transformToValuePair(option);
-                const radioButtonId = `${id}-feedback--${optionValue}`;
+import { useFeedback } from "./useFeedback";
+import { FeedbackType, FeedbackAnswer, FeedbackQuestion } from "./types";
+import { FollowupWizard } from "./FollowupWizard";
+import { MainQuestion } from "./MainQuestion";
 
-                return (
-                    <Fragment key={optionValue}>
-                        <input
-                            className="jkl-feedback__answer-input"
-                            type="radio"
-                            name="feedback"
-                            id={radioButtonId}
-                            value={optionValue}
-                            onChange={() => setValue(optionValue)}
-                            checked={value === optionValue}
-                        />
-                        <label
-                            data-testid={`feedback-${optionValue}`}
-                            className="jkl-feedback__answer-label"
-                            htmlFor={radioButtonId}
-                        >
-                            <Smiley element={optionValue} />
-                            <ScreenReaderOnly>{label}</ScreenReaderOnly>
-                        </label>
-                    </Fragment>
-                );
-            })}
-        </FieldGroup>
-    );
-};
+type MainProps = Omit<ComponentProps<typeof MainQuestion>, "feedbackState">;
+interface Props extends MainProps {
+    onSubmit: (f: FeedbackType) => void;
+    followUp?: {
+        questions: FeedbackQuestion[];
+        onSubmit: (values: FeedbackAnswer[]) => void;
+    };
+}
+
+export const Feedback = ({ onSubmit, followUp, ...mainQuestionProps }: Props) => {
+    const feedbackState = useFeedback(onSubmit);
+    const [showFollowup, setShowFollowup] = useState(!!followUp);
+    const [followupStarted, setFollowupStarted] = useState(false);
 
-export const Feedback: FC<BaseFeedbackProps> = (props) => {
-    return <BaseFeedback {...props} content={<FeedbackContent />} />;
+    useEffect(() => setShowFollowup(!!followUp), [followUp]);
+
+    return (
+        <div className={cn("jkl-feedback")}>
+            {!feedbackState.submitted && <MainQuestion {...mainQuestionProps} feedbackState={feedbackState} />}
+            {feedbackState.submitted && !followupStarted && (
+                <div data-testid="jkl-feedback__feedback-success" className="jkl-feedback__fade-in">
+                    <SuccessMessage title="Takk for tilbakemeldingen!">
+                        Husk at vi ikke kan svare deg her. Lurer du på noe annet ta kontakt med din bankrådgiver.
```

Comment:
```suggestion
                        Husk at vi ikke kan svare deg her. Lurer du på noe annet, ta kontakt med din bankrådgiver.
```
Jeg troor det burde være komma her.